tutorials/dids/cheqd-cosmos-cli/update-and-manage-did-document.md

```diff
@@ -169,147 +63,15 @@ cheqd-noded tx cheqd create-did '{"id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n",\
+cheqd-noded tx cheqd update-did "$(cat diddoc.json)" "FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q==" \
   --from my_account --node http://nodes.testnet.cheqd.network:26657 --chain-id cheqd-testnet-4 --fees 5000000ncheq
 ```
-
-Let's the result will be like:
-
-```
-{"pub_key_base_64":"MnrTheU+vCrN3W+WMvcpBXYBG6D1HrN5usL1zS6W7/k=","pub_key_multibase_58":"",\
-"priv_key_base_64":"FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q=="}
-```
-
-1. Get multibase58 string for inserting it into the DID-doc (`public_key_multibase` field in `verification_method` section)
-
-```
-cheqd-noded debug encoding base64-multibase58 MnrTheU+vCrN3W+WMvcpBXYBG6D1HrN5usL1zS6W7/k=
-```
-
-Response will be:
-
-```
-z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE
-```
-
-1. We need to insert multibase58 string from step 2 as `public_key_multibase` field in `verification_method` section and create a unic identifier for `verification_method`. It should be `<DID>#<some_unic_String>`. Let it be `did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1`. After those preparations, the base DID-doc will look like:
-
-```
-{
-  "id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n",
-  "verification_method": [
-    {
-      "id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1",
-      "type": "Ed25519VerificationKey2020",
-      "controller": "id:cheqd:testnet:zJ5EDiiiKWDyo79n",
-      "public_key_multibase": "z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE"
-    }
-  ],
-  "authentication": [
-    "id:cheqd:testnet:zJ5EDiiiKWDyo79n#key1"
-  ]
-}
-```
-
-And the full command for sending it to the pool will be:
-
-```
-cheqd-noded tx cheqd create-did '{"id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n",\
-                                   "verification_method": [{\
-                                     "id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1",\
-                                     "type": "Ed25519VerificationKey2020",\
-                                     "controller": "id:cheqd:testnet:zJ5EDiiiKWDyo79n",\
-                                     "public_key_multibase": "z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE"\
-                                   }],\
-                                   "authentication": [\
-                                     "id:cheqd:testnet:zJ5EDiiiKWDyo79n#key1"\
-                                   ]\
-                                 }' "id:cheqd:testnet:zJ5EDiiiKWDyo79n#key1" \
-  --ver-key "FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q==" \
-  --from one_of_our_test_account --node https://rpc.testnet.cheqd.network:443 --chain-id cheqd-testnet-4 --fees 50ncheq
-```
-
-1. For checking, that DID was successfully written the next query can be used:
-
-```
-cheqd-noded query cheqd did "did:cheqd:testnet:zJ5EDiiiKWDyo79n" --output json
-```
-
-### Requirements from OS side
```

Comment:
Could you please create a new doc with a full example of creating, updating, requesting DID Doc + CLI installation instructions from this section?